### PR TITLE
apktool: add Japanese translation

### PR DESCRIPTION
**Pull request by Ry0taK**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Translated [apktool](https://github.com/tldr-pages/tldr/blob/ba8ab0fcf067fe8960e586ee74154fe05b778a41/pages/common/apktool.md) to Japanese.

### CODE REVIEW

File: pages.ja/common/apktool.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# apktool
+
+> APKファイルの解析ツール
```
Comment (blueskyson) [inline pages.ja/common/apktool.md:3]:
```suggestion
> APK ファイルの解析ツール
```

File: pages.ja/common/apktool.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+> 詳しくはこちら: <https://ibotpeaches.github.io/Apktool/>.
+
+- APKファイルのデコード
```
Comment (blueskyson) [inline pages.ja/common/apktool.md:6]:
```suggestion
- APK ファイルのデコード
```

File: pages.ja/common/apktool.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+`apktool d {{APKファイルのパス}}`
+
+- 指定したディレクトリからAPKファイルを生成:
```
Comment (blueskyson) [inline pages.ja/common/apktool.md:10]:
```suggestion
- 指定したディレクトリから APK ファイルを生成:
```

**Review by blueskyson — COMMENTED** [review]
Thank you for contribution. Please put spaces between full-width characters and Latin characters.

**Comment by Ry0taK** [comment]
@blueskyson
Thank you for the review! I've applied these suggestions.

**Review by blueskyson — APPROVED** [review]

File: pages.ja/common/apktool.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# apktool
+
+> APK ファイルの解析ツールです。
+> 詳しくはこちら: <https://ibotpeaches.github.io/Apktool/>
```
Comment (navarroaxel) [inline pages.ja/common/apktool.md:4]:
```suggestion
> 詳しくはこちら: <https://ibotpeaches.github.io/Apktool/>.
```

Comment (Ry0taK) [reply pages.ja/common/apktool.md]: Thanks! I've added the period to end of the description.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**